src/Worker.php

```diff
@@ -106,8 +106,11 @@ public function processMessage($msg)
 
                 if ($hash != $payload->pdfHash) {
                     $pdfGenerator = new PdfGenerator($this->curlProxy, $payload->pdfGeneratorUrl, false);
-                    $pdfPath = $pdfGenerator->getPdf($payload->url, $payload->layoutParams->orientation);
-
+                    if (!isset($payload->layoutParams->pageSize)) {
+                        $pdfPath = $pdfGenerator->getPdf($payload->url, $payload->layoutParams->orientation);
```

Comment:
I think, you can remove this line and just keep next line with `pageSize` parameter (in else condition).
After we will keep in mind to deploy this PR https://github.com/CanalTP/MttBundle/pull/158 before this one